src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/oauth/RemoteAPIAjaxControllerTest.java

```diff
@@ -149,18 +142,4 @@ public void getAjaxAPIListTest() {
 		assertEquals("Should have 1 Remote API", 1, response.getDataSource()
 				.size());
 	}
-
-	@Test
-	public void checkAPIStatusTest() {
```

Comment:
Is this test re-written somewhere else?

Reply:
Yes, they are all in the UIRemoteAPIServiceTest